code/style.css

```diff
@@ -1,4 +1,293 @@
+@import url('https://fonts.googleapis.com/css2?family=Bungee&display=swap');
+
+/* GLOBAL STYLES*/ 
+* {padding: 0; margin: 0;}
+
 html {
-  background: #0025ff;
-  color: #fff;
+    background: #222222;
+    color: #E5E5E5;
+    font-family: 'Bungee'; 
+}
+
+body {
+  position:relative;
+  min-height: 100vh;
+}
+
+p {
+  font-family: 'Roboto';
+  font-size: 14px;
+}
+
+/* HEADER & NAVIGATION */
+.flex-container  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  margin-bottom: 1rem;
+}
+
+.logo {
+    margin: 0 2em;
+}
+
+.navigation  {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.navigation a {
+  display: block;
+  padding: 1em;
+  color: white;
+  text-decoration: none;
+  letter-spacing: .45rem;
+}
+
+.navigation a:hover {
+  background: #00ace6;
+}
+
+/* BIG NEWS SECTION */ 
+
+.latest-news-title {
+  font-size: 40px;
+  padding: 0 0 2rem 0;
+}
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  width: auto;
+  height: auto;
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
+  width: 1200px;
+  height: 1000px;
+
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding: 2rem;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  width: 1000px;
+  padding-top: 1rem;
+}
+
+.big-news-image {
+  width: 1000px;
+ }
+
+.big-news-title {
+  font-size: 32px;
+  padding-bottom: 1em;
+}
+
+.big-news-text {
+  font-size: 14px;
+  display: block;
+}
+
+.big-news-text a {
+  font-size:20px;
+  font-family:'Bungee'; 
+  text-decoration: underline;
+}
+
+.big-news-text a:hover {
+  color:#00ace6;
+}
+
+.little-title {
+  padding-top: 2rem;
+  font-size: 12px;
+  color:rgba(126, 126, 126, 0.61);
+}
+
+
+/* OTHER NEWS SECTIONS*/ 
+
+.other-news {
+  display: flex;
+  width: 1200px;
+  flex-wrap: wrap;
+  padding:1rem;
+}
+
+.news-card {
+  width: 380px;
+  height: 400px;
+
+
+}
+
+.other-news-image {
+  width: 340px;
+}
+
+.other-news-title {
+  padding: 1rem 0;
+}
+
+.other-news-text {
+  font-size: 12px;
+  width: 340px;
+}
+
+.other-news a {
+  font-size: 14px;
+  font-family: 'Bungee';
+  text-decoration: underline;
+  display: block;
+  padding-top:1rem;
+}
+
+.other-news a:hover {
+  color:#00ace6;
+}
+
+/* FOOTER */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 8px 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding:1rem;
+}
+
+footer p {
+  font-family: 'Bungee';
+  font-size: 16px;
+  letter-spacing: .10rem;
+  padding:1em;
+}
+
+.project {
+  font-family: 'Roboto';
+  text-align: center;
+  font-size: 10px;
+}
+
+footer a:hover {
+  color:#00ace6;
+}
+
+
+
+
+
+
+/* ******MEDIA QUERIES****** */ 
+
+
+
+
+
+
+/* STYLE FOR TABLET */
+
+@media (max-width: 1259px) {
```

Comment:
**Tablet** 
Big news image has a width of 1000px - it makes the rest of the content move around and you get a scrollbar in the tablet.  

  **Potential solution:**  
  Remove width 1000px. 
  Add:
  {
flex-direction: column; 
width: auto; 
height: auto;
}
 

Reply:
Where exact should I add it in my code? :)

Reply:
In your @media queries (both mobile and tablet)

make class 

.big-new-image {
flex-direction: column; 
width: auto; 
height: auto;
}

Reply:
I get a little big confusing about my classes i big-news section, because it is a lot of boxes there. I think it must be another way to do it.

Reply:
Ah okay! But I think it's the header image that needs to have that code on it. Somehow you need to remove the 1000px width in mobile and tablet mode.

Reply:
By the way I didn't know that you can use height to change the size of logo. It will really helpful for my future projects :)